e2e/tests-dfx/upgrade_check.bash

```diff
@@ -78,3 +78,17 @@ teardown() {
   assert_command dfx canister call hello_backend f '()'
   assert_match "(opt \"\")"
 }
+
+@test "warning when dropping stable variable" {
+  install_asset upgrade
+  dfx_start
+  dfx deploy
+  dfx canister call hello_backend inc '()'
+  jq '.canisters.hello_backend.main="v5.mo"' dfx.json | sponge dfx.json
+  echo yes | (
+  assert_command dfx deploy
+  assert_match "Stable interface compatibility check issued an ERROR"
```

Comment:
With the next Motoko version, this would become a warning and the assertion would need to be adjusted to:
```
assert_match "Stable interface compatibility check issued a WARNING"
```